utils/distance.py

```diff
@@ -7,4 +7,11 @@ def euclidean_distance(first, second):
 
 
 def sum_lists(first, second):
-    return [(float(elemA) - float(elemB)) ** 2 for elemA, elemB in zip(first, second)]
\ No newline at end of file
+    return [(float(elemA) - float(elemB)) ** 2 for elemA, elemB in zip(first, second)]
+	
+	
+def calc_distance(point, point_list):
+	size_point = len(point) - 1
+	point1 = point[0:size_point]
+	return [(euclidean_distance(point1, point2[0:size_point]),point2[size_point]) for point2 in point_list]
```

Comment:
Mano, lembra que se você quiser pegar a lista desde o início, você não precisa colocar a posição inicial `[:size_point]`. Mas colocar ou não, não interfere em nada. Ficou muito bom